lib/utils/normalizeOptions.js

```diff
@@ -73,6 +73,9 @@ function normalizeOptions(compiler, options) {
   options.liveReload =
     typeof options.liveReload !== 'undefined' ? options.liveReload : true;
 
+  options.host =
+    typeof options.host !== 'undefined' ? options.host : 'localhost';
```

Comment:
```suggestion
    typeof options.host !== 'undefined' ? options.host : null;
```

Reply:
`null` won't fix the issue here - https://github.com/webpack/webpack-dev-server/pull/2992#discussion_r571360196 . I'm still unsure about this approach

Reply:
Thanks for updating issue.
My suggestion was based on test here https://github.com/webpack/webpack-dev-server/blob/master/test/server/host-option.test.js

Ideally, https://github.com/webpack/webpack-dev-server/blob/master/test/server/host-option.test.js#L18 and https://github.com/webpack/webpack-dev-server/blob/master/test/server/host-option.test.js#L48 should have same behaviour. But that maybe a breaking change.